x-pack/plugins/ml/server/routes/schemas/data_analytics_schema.ts

```diff
@@ -60,6 +60,14 @@ export const analyticsIdSchema = schema.object({
   analyticsId: schema.string(),
 });
 
+export const analyticsIdIndexSchema = schema.object({
```

Comment:
I think this should be renamed to something like `deleteDataFrameAnalyticsJobSchema` as it is specific to deleting the job.

Reply:
Done in https://github.com/elastic/kibana/pull/66934/commits/ba820793f855ac45c4109ec490bf0cfce1a98fcc